backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pickingslotsClearing/PackingHUsViewFactory.java

```diff
@@ -108,15 +120,15 @@ public WindowId getWindowId()
 	}
 
 	@Override
-	public void setViewsRepository(IViewsRepository viewsRepository)
+	public void setViewsRepository(final IViewsRepository viewsRepository)
 	{
 		this.viewsRepo = viewsRepository;
 	}
 
 	@Override
 	public void put(final IView view)
 	{
-		throw new UnsupportedOperationException();
+		views.put(view.getViewId(), view);
```

Comment:
not sure this OK.
we are storing the PackingHUsView inside the PickingSlotsClearingView

Reply:
removed.